go.mod

```diff
@@ -43,7 +43,7 @@ require (
 	github.com/pseudomuto/protokit v0.2.0
 	github.com/radovskyb/watcher v1.0.2
 	github.com/rotisserie/eris v0.1.1
-	github.com/sam-heilbron/protoc-gen-openapi v0.0.0-20210521134209-c323189e7a9a
+	github.com/sam-heilbron/protoc-gen-openapi v0.0.0-20210719212650-45522763336d
```

Comment:
I think this is fine for now, but if do any additional changes on prrotoc-gen-openapi, we should move it to the solo-io org and add changelog / release versioning to track breaking/other changes. Not super important right now ofc because this is the only repo that uses it.

Reply:
Agreed!